src/Elastic.NLog.Targets/README.md

```diff
@@ -0,0 +1,116 @@
+# Elastic.NLog.Targets
+
+A [NLog](https://nlog-project.org/) target that writes logs directly to [Elasticsearch](https://www.elastic.co/elasticsearch/) or [Elastic Cloud](https://www.elastic.co/cloud/)
+
+## Packages
+
+The .NET assemblies are published to NuGet under the package name [Elastic.NLog.Targets](http://nuget.org/packages/Elastic.NLog.Targets)
+
+## How to use from API
+
+```csharp
+var config = new LoggingConfiguration();
+var elasticTarget = new ElasticsearchTarget("elastic") { Layout = new EcsLayout(), NodesUri = "http://localhost:9200" };
```

Comment:
Specifying the Layout is not necessary here correct? it defaults to this?

Reply:
It was mostly to show that the `ElasticsearchTarget` relies on the `EcsLayout`, and you can configure the `EcsLayout` if you want to override the default output.